src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -31,6 +31,7 @@ public class ScriptHostManager : IScriptHostEnvironment, IDisposable
         private readonly IDisposable _fileEventSubscription;
         private readonly StructuredLogWriter _structuredLogWriter;
         private ScriptHost _currentInstance;
+        private int _hostRestartCount;
```

Comment:
In investigating memory leaks, I wanted to know how many hosts we've created/disposed to correlate with root objects in the DMP. Figured this might be useful info to log as well.